src/components/JoinUs/Perks.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import styled from 'styled-components'
+import remcalc from 'remcalc'
+import breakpoint from 'styled-components-breakpoint'
+import { Row, Col } from '../grid'
+import { Padding } from 'styled-components-spacing'
+import { Paragraph, H2 } from '../Typography'
+import Image from '../Common/Image'
+
+import { Section, ShiftedColumn } from './elements'
+
+const PerksListContainer = styled(ShiftedColumn)`
+  display: grid;
+  row-gap: ${remcalc(24)};
+  grid-template-columns: 1fr 1fr;
+  column-gap: 42px;
+  padding-top: ${remcalc(24)};
+
+  ${breakpoint('smallPhone')`
+    grid-template-columns: 1fr 1fr;
+  `};
+
+  ${breakpoint('smallTablet')`
+    grid-template-columns: 1fr 1fr 1fr;
+  `};
+
+  ${breakpoint('tablet')`
+    grid-template-columns: 1fr 1fr;
+    row-gap: ${remcalc(36)};
+    column-gap: 42px;
```

Comment:
If you compare the Zeplin version of the Join Us page:
### Zeplin
![image](https://user-images.githubusercontent.com/15656538/51605253-708c0480-1f06-11e9-9581-b5620e79af8b.png)

### Coded
![image](https://user-images.githubusercontent.com/15656538/51605205-45091a00-1f06-11e9-8ff3-20fa4e53751a.png)

you can see that this spacing is not quite as designed (48px on desktop, not 42px)

### See Grid documentation in Zeplin
![image](https://user-images.githubusercontent.com/15656538/51605100-f52a5300-1f05-11e9-86c3-4c9f97b76f4e.png)

![image](https://user-images.githubusercontent.com/15656538/51605118-05423280-1f06-11e9-9b0e-684236199173.png)

It's the gutters that are hard-coded to a specific value & then the column widths are dependent on what is left over.

etc. (More breakpoints visible in Zeplin)

Reply:
the whole grid should respect that spacing. I didn't do since I though the grid/gutter was to be reviewed. But I now changed this section to follow the specs